x/pricefeed/simulation/operations/msg.go

```diff
@@ -0,0 +1,149 @@
+package operations
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+
+	"github.com/kava-labs/kava/x/pricefeed"
+	"github.com/kava-labs/kava/x/pricefeed/keeper"
+	"github.com/kava-labs/kava/x/pricefeed/types"
+
+	tmtime "github.com/tendermint/tendermint/types/time"
+)
+
+var (
+	noOpMsg     = simulation.NoOpMsg(pricefeed.ModuleName) // TODO QUESTION what is the purpose of noopmsg?
+	govDenom    = "ukava"                                  // TODO QUESTION do I need these in pricefeed?
```

Comment:
I don't believe pricefeed module will require `govDenom`

Reply:
You are right

Reply:
I will remove it